tests/basic/test_recipe_model_download_consistency.py

```
@@ -0,0 +1,180 @@
+
+
+def _load_documents(path: Path) -> List[Any]:
+    return [doc for doc in yaml.safe_load_all(path.read_text()) if doc is not None]
+
+
+def _repo_ids_from_flags(text: str) -> Set[str]:
+    """Extract checkpoint ids from ``--model <id>`` style flags in a script."""
+    found: Set[str] = set()
+    for line in text.splitlines():
+        tokens = line.replace("\\", " ").split()
+        for flag, value in zip(tokens, tokens[1:]):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)